package.json

```diff
@@ -94,6 +94,7 @@
     "express": "^4.21.2",
     "gcp-metadata": "^5.3.0",
     "js-yaml": "^4.1.0",
+    "kerberos": "^2.2.1",
```

Comment:
https://parsley.mongodb.com/evergreen/mongo_node_driver_next_rhel80_large_Node22_test_auth_kerberos_patch_5f4500b85cded02f6158f4eb043bc6c926bd92c0_67ab9169bbbb0000078b9aed_25_02_11_18_05_31/0/task?bookmarks=0,1036&shareLine=120

Dunno why kerberos claims to be missing but we track all our other peers in our devDeps.